designs/administrator/olm/global-operator-all-namespace-operands/index.md

```diff
@@ -0,0 +1,41 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Display All Namespace Operands for Global Operators
+
+Today in the console, when a user clicks on an operator that manages “All namespaces” (globally installed) from the “All Projects” view of the Installed Operators page, they are taken to a the details page of where that particular operator is installed. The project selector switches from "All projects" to the specific project name where the operator was installed.
+
+This can be disorienting then to look at the lists of custom resource instances and see them all blank or not contain all the instances you may expect, since the lists are showing instances only in the currently selected project (the install namespace) and not across all namespaces the operator is available in.
+
+It is likely that making use of the new Operator resource will improve this experience as that resource is cluster-scoped, in which case the operator view would be across all namespaces and would show all operand instances in these lists, though that may still be some releases away.
+
+In the meantime the console can make the default view for globally installed operators to show all instances across all namespaces, with a way to still see only the instances in the selected namespace in the project selector.
+
+## Global Operators
+
+![Global operator list view](img/1-1-global-list.png)
+- User opens a globally available operator.
+
+![Global operator operands 1](img/1-2-global-allOny.png)
```

Comment:
This is a nit and honestly out of scope for this PR but I wonder why we need to show the "NS" badge for the namespace column. Isn't the whole point of the badge to make the differences more scannable? In this case it forces a line break and a bit of visual clutter. Would be nice if we could remove here but not sure what the impact across the cluster would be.